style.css

```
@@ -1,7 +1,164 @@
+/* Root Styles from Style-Guide.md */
+:root {
+  --pale-blue: hsl(225, 100%, 94%);
+  --bright-blue: hsl(245, 75%, 52%);
+
+  --very-pale-blue: hsl(225, 100%, 98%);
+  --desaturated-blue: hsl(224, 23%, 55%);
+  --dark-blue: hsl(223, 47%, 23%);
+
+  /* Typography and Spacing */
+  --font-family: "Red Hat Display", sans-serif;
+  --font-size-body: 16px;
+
+  --padding: 1.5rem;
+  --border-radius: 12px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+
+  flex-direction: column;
+
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  padding: var(--padding);
+  font-family: var(--font-family);
+  font-size: var(--font-size-body);
+  background: url(./images/pattern-background-mobile.svg) center center
+    no-repeat;
+  background-size: cover;
+  background-color: var(--pale-blue);
+  margin: 0;
+  overflow-y: auto;
+}
+
+.container {
+  background: #fff;
+  border-radius: var(--border-radius);
+  box-shadow: 0 10px 20px rgba(0, 0, 0, 0.1);
+  text-align: center;
+  max-width: 375px;
+  width: 100%;
+  overflow: hidden;
+  font-weight: 500px;
+}
+
+.image-container {
+  width: 100%;
+}
+
+.content {
+  padding: var(--padding);
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.header {
+  color: var(--dark-blue);
+  font-weight: 900;
+  font-size: 22px;
+}
+
+p {
+  color: var(--desaturated-blue);
+  font-size: 16px;
+  margin-bottom: 20px;
+}
+
+.plan {
+  background: var(--very-pale-blue);
+  padding: 1rem;
+  border-radius: var(--border-radius);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 20px 0;
+}
+
+.plan img {
+  width: 40px;
+}
+
+.plan-text {
+  flex-grow: 1;
+  margin-left: 10px;
+  text-align: left;
+}
+
+.plan-text span {
+  display: block;
+  color: var(--dark-blue);
+  font-weight: 700;
+  font-size: 16px;
+}
+
+.plan-text small {
+  color: var(--desaturated-blue);
+  font-size: 14px;
+}
+
+.plan a {
+  color: var(--bright-blue);
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.btn {
+  display: block;
+  background: var(--bright-blue);
+  color: white;
+  padding: 0.725rem 1rem;
+  border-radius: var(--border-radius);
+  text-decoration: none;
+  font-weight: 700;
+  box-shadow: 0 5px 15px rgba(0, 0, 255, 0.3);
+  margin-bottom: 15px;
+}
+
+.btn:hover {
+  background: hsl(245, 80%, 60%);
+}
+
+.cancel {
+  display: block;
+  margin-top: 1rem;
```

chnage to margin-bottom instead to match with spacing on design
```suggestion
  margin-bottom: 1rem;
```

---

Done